src/hmem_cuda_gdrcopy.c

```diff
@@ -410,4 +449,18 @@ int cuda_gdrcopy_dev_unregister(uint64_t handle)
 	return FI_SUCCESS;
 }
 
+ssize_t ofi_gdrcopy_to_cuda_iov(uint64_t handle, const struct iovec *iov,
+                                size_t iov_count, uint64_t iov_offset,
+								const void *hostptr, size_t len)
+{
+	return -FI_EOPNOTSUPP;
```

Comment:
I think we usually return `-FI_ENOSYS` for this

Reply:
Right. Addressed.